services/skus/controllers.go

```diff
@@ -703,6 +703,10 @@ func GetOrderCreds(service *Service) handlers.AppHandler {
 			)
 		}
 
+		l := logging.Logger(ctx, "skus").With().
+			Str("func", "GetOrderCreds").
+			Logger()
```

Comment:
nit: not sure I like this formatting. Do we need it? Our screens seem to fit this.

```suggestion
		l := logging.Logger(ctx, "skus").With().Str("func", "GetOrderCreds").Logger()
```

It might also make sense to make the logger be `lg`, and then if we need a more narrowly-focused sublogger, that could be `l`.